src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -10,14 +10,15 @@ import {uploadFile} from '../../../service/file';
 
 const useGroupSetting = ({navigation, route}) => {
   const [groupChatState] = React.useContext(Context).groupChat;
+
   const [channelState] = React.useContext(Context).channel;
   const [profile] = React.useContext(Context).profile;
-  const {participants} = groupChatState;
+  const [participants, setParticipants] = React.useState([]);
   const {channel} = channelState;
   const [groupName, setGroupName] = React.useState(
-    getChatName(route.params.username, profile.myProfile.username) || 'Set Group Name'
+    getChatName(route.params.name, profile.myProfile.username) || 'Set Group Name'
   );
-  const [countUser] = React.useState(Object.entries(participants).length);
+  const [countUser, setCountUser] = React.useState(0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `participants` state is now initialized as an empty array and the `countUser` state is initialized as 0. This could potentially lead to issues if these states are used before they are updated in the `useEffect` hook. It would be better to initialize them with the actual initial values.
```diff
-   const [participants, setParticipants] = React.useState([]);
-   const [countUser, setCountUser] = React.useState(0);
+   const {participants} = groupChatState;
+   const [countUser] = React.useState(Object.entries(participants).length);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->